src/frontend/main.js

```diff
@@ -87,12 +88,17 @@ $(function() {
     };
 
     const removeBookmark = function() {
-        const bookmark = results.find('.selected');
-        const id = bookmark.data('id');
-        bookmarksCollection.remove(id, function() {
-            selectNext();
-            bookmark.remove();
-            loadBookmarks();
+        alerts.confirmate('Are you sure? You will not be able to recover this bookmark!', function(result) {
```

Comment:
not sure, it already uses very clear interface, additionally it will be like "exceptions control flow" which is bad.